knowledge-store-investigation/push.sh

```diff
@@ -16,4 +16,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-fsoc solution push --directory ${USER}malware --wait --tag=base -v
```

Comment:
Why remove --wait and -v?

Reply:
needs not only `--wait` but also `subscribe` (otherwise pointless to wait). But `--subscribe` fails, needs to be fixed in fsoc.